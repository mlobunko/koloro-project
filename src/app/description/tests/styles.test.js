import React from "react";
import { shallow } from "enzyme";
import "enzyme-to-json";

import {
  StyledBoldParagraph,
  StyledH1,
  StyledItalicText,
  StyledStandardParagraph
} from "../styles";

describe("StyledBoldParagraph", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledBoldParagraph landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledBoldParagraph landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledH1", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledH1 landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledH1 landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledItalicText", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledItalicText />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledStandardParagraph", () => {
  it("renders correctly, landscape true", () => {
    const landscape = true;
    const wrapper = shallow(<StyledStandardParagraph landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders correctly, landscape false", () => {
    const landscape = false;
    const wrapper = shallow(<StyledStandardParagraph landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});
