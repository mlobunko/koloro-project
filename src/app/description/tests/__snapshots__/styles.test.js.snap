// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledBoldParagraph renders correctly, landscape false 1`] = `
<div
  className="sc-bdVaJa fGCmPb"
/>
`;

exports[`StyledBoldParagraph renders correctly, landscape true 1`] = `
<div
  className="sc-bdVaJa kyzRwg"
/>
`;

exports[`StyledH1 renders correctly, landscape false 1`] = `
<h1
  className="sc-bwzfXH fTEPwo"
/>
`;

exports[`StyledH1 renders correctly, landscape true 1`] = `
<h1
  className="sc-bwzfXH eLVHlA"
/>
`;

exports[`StyledItalicText renders correctly 1`] = `
<span
  className="sc-htpNat hQpdAy"
/>
`;

exports[`StyledStandardParagraph renders correctly, landscape false 1`] = `
<div
  className="sc-bxivhb dFubON"
/>
`;

exports[`StyledStandardParagraph renders correctly, landscape true 1`] = `
<div
  className="sc-bxivhb ifQbED"
/>
`;
